### PR TITLE
aki day 28 code

### PR DESCRIPTION
**Pull request by disandroa**
## Checklist

- [x] You have included a thumbnail image
- [x] You have created a new Markdown file to describe your map
- [x] You have put _at least_ your name and the map day in the pull request

### CODE REVIEW

File: _maps_2024/28-aki.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+day: 28
+creator: Akira Di Sandro
+title: The Great Lakes
+thumbnail: /assets/thumbnails/2024/28-BluePlanet.png
```
Comment (mjumbewu) [inline _maps_2024/28-aki.md:5]: Looks like your file is named "28-Aki-BluePlanet.png". You'll either need to change the name of the file or update the reference here and below (line 15).
```suggestion
thumbnail: /assets/thumbnails/2024/28-Aki-BluePlanet.png
```

File: _maps_2024/28-aki.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day28.R
+---
+
+I grew up in the Midwest, so I wanted to showcase the Great Lakes for today's map challenge. With it also being Thanksgiving (a holiday that whitewashes this country's genocidal history), I wanted to acknowledge the people that lived on the land before colonization.
+
+Finding and mapping data regarding where Indigenous Americans lived proved to be much more challenging than expected. I pulled information from multiple sources and estimated locations of different tribes to the best of my ability based on available information.
+
+Check out this [cool website](https://native-land.ca/) that visualizes indigenous peoples around the world.
+
+![Screenshot of map](assets/thumbnails/2024/28-BluePlanet.png)
```
Comment (mjumbewu) [inline _maps_2024/28-aki.md:15]:
```suggestion
![Screenshot of map](assets/thumbnails/2024/28-Aki-BluePlanet.png)
```

**Review by mjumbewu — CHANGES_REQUESTED** [review]
:heart: A well-timed contribution (regarding the time of the year, not time of day -- isn't it 3AM? I'm in South Africa, so I have an excuse).

File: _maps_2024/28-aki.md (hunk 2 above)
Comment (disandroa) [reply _maps_2024/28-aki.md:15]: thank you for catching that! clearly finishing a map at 3 am took its toll.....😅

**Comment by disandroa** [comment]
adding day 29 materials. i'm not sure if html was the best way to upload this! pls lmk if i should just take a screen shot instead

File: _maps_2024/29-aki.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+day: 29
+creator: Akira Di Sandro
+title: Overture Maps
+thumbnail: /assets/thumbnails/2024/29-Aki-Overture.html
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day29.R
+---
+
```
Comment (mjumbewu) [inline _maps_2024/29-aki.md:8]: **suggestion:** You'll still have to provide a thumbnail image (something to show on the front page -- note that I refer to a PNG file that doesn't yet exist below, so you'd have to create it), but the best thing to do might be to create that image, and then link to the HTML page for the interactive bit:
```suggestion
thumbnail: /assets/thumbnails/2024/29-Aki-Overture.png
source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day29.R
---

View the [interactive map](assets/thumbnails/2024/29-Aki-Overture.html) of building classifications in Firenze, Italy.

![Screenshot of map](assets/thumbnails/2024/29-Aki-Overture.png)
```

**Review by mjumbewu — CHANGES_REQUESTED** [review]
Check out the notes below

File: _maps_2024/29-aki.md (hunk 3 above)
Comment (disandroa) [reply _maps_2024/29-aki.md:8]: thank you! Just added the png as well

File: _maps_2024/29-aki.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+day: 29
+creator: Akira Di Sandro
+title: Overture Maps
+thumbnail: /assets/thumbnails/2024/29-Aki-Overture.png
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day29.R
+---
+
+View the [interactive map](assets/thumbnails/2024/29-Aki-Overture.html) of building classifications in Firenze, Italy.
+
+![Screenshot of map](assets/thumbnails/2024/29-Aki-Overture.png)
```
Comment (mjumbewu) [inline _maps_2024/29-aki.md:11]: :+1: 

Note that you could also use something like the [`iframe`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) tag to embed the map directly on this page; something like:

```html
<iframe
  title="Building classifications in Firenze, Italy"
  src="assets/thumbnails/2024/29-Aki-Overture.html"></iframe>
```

Getting the sizing right on iframes can sometimes be tricky. I don't remember what the "i" in "iframe" stands for :shrug:. You can submit a new PR if you'd like to try it out.

**Review by mjumbewu — APPROVED** [review]
LGTM! If you want, you can try an iframe embed for your interactive map (see below)

File: _maps_2024/29-aki.md (hunk 4 above)
Comment (disandroa) [reply _maps_2024/29-aki.md:11]: I think I will submit as is right now. will try to clean this up when I have more time🥲

File: _maps_2024/13-aki.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+day: 13
+creator: Akira Di Sandro
+title: New Tool
+thumbnail: /assets/thumbnails/2024/13-Aki-NewTool.gif
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day13.R
+---
+
+I hadn't given tmap animations a try yet, so here's my first shot at it!
```
Comment (mjumbewu) [inline _maps_2024/13-aki.md:9]: Adding the image into the description.

```suggestion
I hadn't given tmap animations a try yet, so here's my first shot at it!

![Animated map of Asian population in Philadelphia](assets/thumbnails/2024/13-Aki-NewTool.gif)
```

File: _maps_2024/26-aki.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+day: 26
+creator: Akira Di Sandro
+title: Map Projections
+thumbnail: /assets/thumbnails/2024/26-Aki-MapProjections.png
+source: https://github.com/disandroa/30DayMappingChallenge2024/blob/main/scripts/Day26.R
+---
+
+Very similar to the commonly seen Mercator Projection, the Van der Grinten projection is known to be aesthetically pleasing and generally maintains recognizable shapes. Though the areas are minimally distorted near the equator, the distortion gets much worse as one approaches the poles.
+
+Learn more about the Van der Grinten projection [here](https://en.wikipedia.org/wiki/Van_der_Grinten_projection).
```
Comment (mjumbewu) [inline _maps_2024/26-aki.md:11]: Adding the image into the description.

```suggestion
Learn more about the Van der Grinten projection [here](https://en.wikipedia.org/wiki/Van_der_Grinten_projection).

![A world map in the Van der Grinten projection](assets/thumbnails/2024/26-Aki-MapProjections.png)
```

**Review by mjumbewu — APPROVED** [review]
Made a couple changes, but LGTM!

**Pull request merged**